internal/proto/internal.proto

```diff
@@ -67,4 +67,48 @@ message PrContents {
 
   minder.v1.PullRequest pr = 1;
   repeated File files = 2;
-}
\ No newline at end of file
+}
+
+message SelectorProvider {
+  // the name of the provider, e.g. github-app-jakubtestorg
+  string name = 1;
+  // the class of the provider, e.g. github-app
+  string class = 2;
+}
+
+message SelectorRepository {
+  // the full name of the repository, e.g. stacklok/minder
+  string name = 1;
+  // the provider of the repository
+  SelectorProvider provider = 2;
+
+  // is_fork is true if the repository is a fork, nil if "don't know" or rather
+  // not applicable to this provider
+  optional bool is_fork = 3;
+  // is_private is true if the repository is private, nil if "don't know" or rather
+  // not applicable to this provider
+  optional bool is_private = 4;
```

Comment:
Do we want to include the ["Custom properties" key/values](https://github.blog/changelog/2023-10-12-github-repository-custom-properties-beta/) that GitHub supports under a name like "tags", "labels", or "properties"?

Reply:
yes, I wanted to do that in a follow-up but perhaps it makes sense to include the k-v property already to make the review easier in the sense that we'd see the whole message.

Reply:
I added the properties to the message, filling them in would be done in a follow-up